src/DynamoCoreWpf/UI/Prompts/ChangeScaleFactorPrompt.xaml.cs

```diff
@@ -12,16 +14,43 @@ namespace Dynamo.Prompts
     /// </summary>
     public partial class ChangeScaleFactorPrompt : Window
     {
+        public enum Size
+        {
+            Small = 0,
+            Medium = 1,
+            Large = 2,
+            ExtraLarge = 3
+        }
+
+        public Size size { get; set; }
```

Comment:
All property and method names start with Uppercase so you can name this `ScaleSize` or something like that and you may also want to log this along with the range in the console.